### PR TITLE
add password visibilty toggle to onboarding

### PR DESCRIPTION
**Pull request by luclu**
closes: https://github.com/ethereum/mist/issues/554

### CODE REVIEW

File: interface/client/templates/popupWindows/onboardingScreen.js [hunk 1]
```diff
@@ -322,6 +322,18 @@ The onboardingScreen password template
 @constructor
 */
 
+Template['popupWindows_onboardingScreen_password'].helpers({
+    /**
+    Show password
+
+    @method showPassword
+    */
+    'showPassword': function() {
```
Comment (hiddentao) [inline interface/client/templates/popupWindows/onboardingScreen.js]: would be better to call this `passwordInputType` or something similar.

Comment (luclu) [inline interface/client/templates/popupWindows/onboardingScreen.js]: I copied this helper function from `popupWindows_onboardingScreen_importAccount` helpers.
Would you change that one too or do I miss your argument?

Comment (frozeman) [inline interface/client/templates/popupWindows/onboardingScreen.js]: i would also suggest changing that function helper name to `passwordInputType`

File: interface/client/templates/popupWindows/onboardingScreen.html [hunk 2]
```diff
@@ -178,8 +178,11 @@ <h2 style="display: block; text-align: center;">{{i18n "mist.popupWindows.onboar
                 <span class="dapp-identicon dapp-icon-loading"></span>
                 <h2 style="display: block; text-align: center;">{{i18n "mist.popupWindows.onboarding.creating"}}</h2>
             {{else}}
-                <input type="password" placeholder="{{i18n 'mist.popupWindows.requestAccount.enterPassword'}}" class="password {{#if TemplateVar.get 'passwordsMismatch'}}dapp-error{{/if}}">
-                <input type="password" placeholder="{{i18n 'mist.popupWindows.requestAccount.repeatPassword'}}" class="password-repeat {{#if TemplateVar.get 'passwordsMismatch'}}dapp-error{{/if}}">
+                <input type="{{showPassword}}" placeholder="{{i18n 'mist.popupWindows.requestAccount.enterPassword'}}" class="password {{#if TemplateVar.get 'passwordsMismatch'}}dapp-error{{/if}}">
```
Comment (frozeman) [inline interface/client/templates/popupWindows/onboardingScreen.html]: And here as well: `{{passwordInputType}}`

File: interface/client/templates/popupWindows/onboardingScreen.html [hunk 3]
```diff
@@ -178,8 +178,11 @@ <h2 style="display: block; text-align: center;">{{i18n "mist.popupWindows.onboar
                 <span class="dapp-identicon dapp-icon-loading"></span>
                 <h2 style="display: block; text-align: center;">{{i18n "mist.popupWindows.onboarding.creating"}}</h2>
             {{else}}
-                <input type="password" placeholder="{{i18n 'mist.popupWindows.requestAccount.enterPassword'}}" class="password {{#if TemplateVar.get 'passwordsMismatch'}}dapp-error{{/if}}">
-                <input type="password" placeholder="{{i18n 'mist.popupWindows.requestAccount.repeatPassword'}}" class="password-repeat {{#if TemplateVar.get 'passwordsMismatch'}}dapp-error{{/if}}">
+                <input type="{{passwordInputType}}" placeholder="{{i18n 'mist.popupWindows.requestAccount.enterPassword'}}" class="password {{#if TemplateVar.get 'passwordsMismatch'}}dapp-error{{/if}}">
+                <input type="{{passwordInputType}}" placeholder="{{i18n 'mist.popupWindows.requestAccount.repeatPassword'}}" class="password-repeat {{#if TemplateVar.get 'passwordsMismatch'}}dapp-error{{/if}}">
+                <br><br>
+                <input type="checkbox" id="show-password" name="elements_input_bool" class="abi-input">
```
Comment (frozeman) [inline interface/client/templates/popupWindows/onboardingScreen.html]: I'm strongly against using ids, as with the template scope and classes we can simply avoid using those.
Please change it to a class :)

**Comment by alexvandesande** [comment]
Superseded by this: https://github.com/ethereum/mist/pull/909

**Pull request closed without merging**